src/dataexplorer/dataexplorer-application/src/components/MinePasienter/MinePasienter.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Dashboard = () => {
```

Comment:
```suggestion
const MinePasienter = () => {
```
I would prefer the component to be named the same as the file and don't have duplicated componentnames

Reply:
Should we rename `MinePasienter` to `MyPatients`?

Reply:
Nice catch. I copied the intially empty Dashboard component to create the two others, so didnt realize this.